src/auth/Login.js

```diff
@@ -0,0 +1,96 @@
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+import BaseUrl from '../redux/base_url';
+
+export default function Login() {
+  const navigate = useNavigate();
+  const fldClass = 'block w-full px-4 py-2 mt-2 text-clrPrime bg-white border rounded-md focus:border-clrPrime focus:ring-clrPrime focus:outline-none focus:ring focus:ring-opacity-20';
+  const lblClass = 'block text-sm font-semibold text-gray-800';
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const name = e.target[0].value;
+    const password = e.target[1].value;
+    await axios.post(`${BaseUrl}users/sign_in`, {
+      user: {
+        name,
+        password,
+      },
+    }).then((response) => {
+      const token = response.headers.authorization;
+      if (token && token !== '') {
+        localStorage.setItem('token', token);
+        navigate('/freelancers');
+      }
+    }).catch((err) => {
+      document.getElementById('msg').textContent = `${err.response.data}!`;
+      e.target[0].value = '';
+      e.target[1].value = '';
```

Comment:
:red_circle: Following the [KISS](https://medium.com/infancyit/kiss-keep-it-simple-smart-javascript-habits-b019ce14b5ec) principle, I think you can create a function to encapsulate this operation This will keep your code cleaner and more maintainable. Since all your input fields are text fields You can for example create a `resetFormFields` function as follow

```
const resetFormFields = (fields) => {
   fields.forEach(element => element.value='');
}
```
Usage :point_down: 

`
resetFormFields(e.target)`

Reply:
@bienvenuushindi  thank you for pointing out this, I will fix it right now.

Reply:
done!